### PR TITLE
Github actions experiment

### PR DESCRIPTION
**Pull request by nabokihms**
Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>

### CODE REVIEW

**Comment by nabokihms** [comment]
I haven't managed to add flant_iam tests yet, but I'm on the way. This fact shouldn't block the merging of this PR.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    env:
+      GOFLAGS: -mod=readonly
+    strategy:
+      max-parallel: 6
+      fail-fast: false
```
Comment (shvgn) [inline .github/workflows/ci.yaml:17]: why fail-fast is false?

**Review by shvgn — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (nabokihms) [reply .github/workflows/ci.yaml:17]: If a job for one plugin fails, other jobs will immediately fail as well. I thought that this is not the behavior we want. For us, it is better to see all errors in all plugins.

Comment (shvgn) [reply .github/workflows/ci.yaml:17]: got it

**Pull request merged**